blog/2025/02/this-month-in-julia-world.md

```diff
@@ -0,0 +1,72 @@
+* [PR #57049](https://github.com/JuliaLang/julia/pull/57049) introduces LLVM-based feature detection for CPU parsing.
+* The difficulty of writing multithreaded code, and footguns related to the difference between thread and task, the danger of moving tasks between threads, and the subleties of properly setting up thread- or task-local storage come up often. This is [a recent good discussion of the topic](https://discourse.julialang.org/t/sharp-edge-with-threads-threadid-and-task-migration/124550). It reminds me also of the [recently merged](https://github.com/JuliaLang/julia/pull/55793) `perprocess` / `perthread` / `pertask` constructs which help manage some of these subtleties.
+* An interesting suggestion for having [julia automatically logging to the terminal to current state of progress through a script](https://github.com/JuliaLang/julia/pull/57137).
+* [Unicode v16 is now supported](https://github.com/JuliaLang/julia/pull/56925) in Julia source files. We get access to "face with bags under eyes" but there is still now superscript “q”.
+
+In search of contributors and new maintainers (specify novice/moderate/expert and internals/domain background necessary):
+
+* The Center for Quantum Networks is running [many bounties and minigrants](https://github.com/QuantumSavory/.github/blob/main/BUG_BOUNTIES.md) on a variety of support packages for quantum information science.
+* [SciML is running a large set of funded small projects](https://sciml.ai/small_grants/) – a great opportunity to contribute to the community and be paid for it.
+* ChainRules, a foundational package to the Julia autodiff effort, is looking for a [new primary maintainer](https://discourse.julialang.org/t/chainrules-project-looking-for-a-new-primary-maintainer/115636).
+
+Ecosystem Fora, Maintenance, and Colab Promises (Slack/Zulip/Discourse/Github):
+
+* Speculator.jl is a [fun new tool to speculative precompile functions](https://discourse.julialang.org/t/speculator-jl-reduce-latency-through-speculative-compilation/124344) (in the background) to reduce latency (also in interactive dev use).
+* [Makie 0.22 is now out](https://makie.org/website/blogposts/v0.22/), with further improvements to geometry primitives and 3D capabilities.
+* [BorrowChecker.jl](https://github.com/MilesCranmer/BorrowChecker.jl) is a new package that introduces Rust-like ownership and borrowing semantics to Julia, aiming to improve memory safety. Related to this earlier [discourse discussion](https://discourse.julialang.org/t/package-for-rust-like-borrow-checker-in-julia/124442).
+* [Statistics.median](https://github.com/JuliaArrays/StaticArrays.jl/pull/973) is now available for StaticArrays, providing efficient median computation via bitonic sorting.
+* AlgebraOfGraphics.jl (inspired by R’s grammar of graphics ggplot, built on Makie) is working surprisingly well with [sophisticated edge-case composition of plots](https://julialang.slack.com/archives/C8RQUU2KH/p1737034714609659).
```

Comment:
How about linking https://aog.makie.org/stable/gallery/gallery/scales/split_scales_facet/#Split-scales-across-facets instead? The slack link will be dead in less than 90 days

Reply:
thanks for flagging this, it will be fixed shortly!